docs/source/python_api/mlflow.llm.rst

```diff
@@ -1,6 +0,0 @@
-mlflow.llm
```

Comment:
Very odd to have an `mlflow.llm` docs page with one deprecated API. Let's remove it.